kafka-streams/src/main/java/StreamsFilterBalance2.java

```diff
@@ -94,9 +96,17 @@ public String apply(String jsonValue) {
                 boolean anomalyFound = false;
                 String anomalyType = "";
 
+
                 if (accounts.containsKey(key)) {
+                    List<Double> last100 = getLast100Trxns(accounts, 1000);
```

Comment:
the method is called getLast100Trxns but 1000 is being sent. Why?

Reply:
I tested with 100 window period and had named the variables as 'Last100', but could as use a window size of 1000 without affecting throughput too much.